src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildWithLibraryTests.cs

```diff
@@ -87,6 +95,7 @@ public void DotNetBuildLibrary (bool isRelease, bool duplicateAar, bool useDesig
 			using (var aar = ZipHelper.OpenZip (aarPath)) {
 				aar.AssertContainsEntry (aarPath, "assets/bar/bar.txt");
 				aar.AssertEntryEquals (aarPath, "proguard.txt", "# LibraryC");
+				aar.AssertContainsEntry (aarPath, "AndroidManifest.xml");
```

Comment:
Should we also assert that the final *app* contains the `<queries/>` element from `libC`?

Reply:
It is checked later on in the test when the appliction is built.